login.html

```diff
@@ -0,0 +1,119 @@
+---
+layout: index.njk 
+title: Login
+---
+
+<h1>{{ title }}</h1>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+div{
+    display:block;
+    align-items: center;
+    justify-content: center;
+    /*color: white; /*overall text colour*/
+    
+}
+    
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: black;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/* Add padding to containers */
+.container {
+  padding: 16px;
+  background-color: white;
+}
+
+/* Full-width input fields */
+input[type=text], input[type=password] {
+  width: 100%;
+  padding: 15px;
+  margin: 5px 0 22px 0;
+  display: inline-block;
+  border: none;
+  background: #f1f1f1;
+}
+
+input[type=text]:focus, input[type=password]:focus {
+  background-color: #ddd;
+  outline: none;
+}
+
+/* Overwrite default styles of hr */
+hr {
+  border: 1px solid #f1f1f1;
+  margin-bottom: 25px;
+}
+
+/* Set a style for the submit button */
+.registerbtn {
+  background-color: #04AA6D;
+  color: white;
+  padding: 16px 20px;
+  margin: 8px 0;
+  border: none;
+  cursor: pointer;
+  width: 100%;
+  opacity: 0.9;
+}
+
+.registerbtn:hover {
+  opacity: 1;
+}
+
+/* Add a blue text color to links */
+a {
+  color: dodgerblue;
+}
+
+/* Set a grey background color and center the text of the "sign in" section */
+.signin {
+  background-color: #f1f1f1;
+  text-align: center;
+}
+</style>
+</head>
+<body>
+
+<div>
+  <div class="container">
+    <label for="username"><b>Username</b></label>
+    <input type="text" placeholder="Enter Username" name="username" id="username" autocomplete="off" required>
+    <hr>
+
+    <button onclick="checkString()" class="registerbtn">Register</button>
+  </div>
+  
+  <div class="container signin">
+    <p>Don't have an account? <a href="/register">Sign in</a>.</p>
+  </div>
+</div>
+
+<script>
+
+  function checkString() {
```

Comment:
Nice work implementing a different style of fetch than we have through this semester